gcp/rakefiles/entrypoint.rake

```diff
@@ -28,7 +29,14 @@ task :default => :deploy
 
 task :set_vars do
   Vars.set_vars(@env, @project_type)
-  @secrets = Secrets.set_secrets()
+  @secrets = Secrets.collect_secrets()
+  File.open("#{@dot_config_path}/compose.env", 'w+') do |file|
+    file.write($compose_env.join("\n"))
```

Comment:
Instead of building `$compose_env` every time we mentioned a `TF_VAR_` variable, how about `:set_vars` grabs all environment variables that start with `TF_VAR_` and writes them into `compose.env`?

Reply:
Great idea!